### PR TITLE
Removed local installation of the ALSA config file

### PR DESCRIPTION
**Pull request by radujipa**
Previously, the ALSA config file was installed both under /etc/
and /etc/skel. The latter meant that the config was installed for
every user under ~/.asoundrc, however, no changes are expected to
be in user space. The reason for this was a need to change some
options system wide (max_dB, see kano-settings).

### CODE REVIEW

**Comment by radujipa** [comment]
[[P1] As a CKC user, I do not want audio distortions when using the CKC speaker at full volume](https://trello.com/c/AnP58vC7/1331-p1-as-a-ckc-user-i-do-not-want-audio-distortions-when-using-the-ckc-speaker-at-full-volume)

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+kano-desktop (3.12.0-0) unstable; urgency=low
+
+  * Removed local installation of the ALSA config file
+
+ -- Team Kano <dev@kano.me>  Mon, 31 Jul 2017 17:38:00 +0100
```
Comment (tombettany) [inline debian/changelog]: Date isn't quite right here...

File: debian/kano-desktop.install [hunk 2]
```diff
@@ -1,11 +1,9 @@
-#!/usr/bin/dh-exec
```
Comment (skarbat) [inline debian/kano-desktop.install]: Probably also remove the execution bit from the file if not already, but no big deal

Comment (radujipa) [reply debian/kano-desktop.install]: Good point! I completely forgot about that part 👍

**Pull request merged**